src/IsoCodes/Bsn.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace IsoCodes;
+
+/**
+ * Class Bsn: In the Netherlands, the citizen service number (BSN) is a unique personal number
```

Comment:
Just remove the `Class Bsn: `, this is useless, we know that the class is named `Bsn` ! :wink: